docker/php-7.1/Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM php:7.1-cli-alpine3.9
```

Comment:
why not 7.1-cli-alpine3.10 ?

Reply:
I didn't notice it was available. I'll update.

Reply:
```suggestion
FROM php:7.1-cli-alpine3.10
```